components/brave_rewards/browser/publisher_info_database.cc

```diff
@@ -491,6 +492,111 @@ bool PublisherInfoDatabase::InsertOrUpdateActivityInfos(
   return transaction.Commit();
 }
 
+uint64_t PublisherInfoDatabase::GetLastReconcileDate() {
+  DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
+
+  bool initialized = Init();
+  DCHECK(initialized);
+  if (!initialized) {
+    return 0;
+  }
+
+  std::string query =
+      "SELECT DISTINCT MAX(date) FROM contribution_info WHERE category = 2";
```

Comment:
let's pass category via param in so like `category = ?`. This way we don't have magic number.

Reply:
Fixed